src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -1162,6 +1162,15 @@ protected function getColumnSqlDefinition(Column $column)
         $def .= $column->getCollation() ? ' COLLATE ' . $column->getCollation() : '';
         $def .= (!$column->isSigned() && isset($this->signedColumnTypes[$column->getType()])) ? ' unsigned' : '';
         $def .= $column->isNull() ? ' NULL' : ' NOT NULL';
+
+        if (
+            version_compare($this->getAttribute(\PDO::ATTR_SERVER_VERSION), '8') > -1
```

Comment:
Stickler-CI putting this on a newline is probably correct (as it follows PSR-12), but I had purposely not so that phpcs would pass. Not sure how to proceed to get a green build.